src/App.tsx

```diff
@@ -1,39 +1,50 @@
 import React from 'react';
 import './App.scss';
+import { time } from 'console';
```

Comment:
Remove it
![image](https://user-images.githubusercontent.com/91411692/231386076-aeeb0b2f-c7ab-41f1-908d-5d5bf8a95103.png)